src/main/java/com/mate/fruitshop/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package com.mate.fruitshop.db;
+
+import com.mate.fruitshop.model.FruitEntry;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<FruitEntry> FRUITS = new ArrayList<>();

```

Comment:
this is not a magic number or something like this
the field can be just static final and not a constant (if it is not a "magic" data)
```suggestion
    public static final List<FruitEntry> fruits = new ArrayList<>();
```